flavors/standard-EXASOL-8.0.0/flavor_base/flavor_base_deps_r/packages/cran_packages

```diff
@@ -1,3 +1,5 @@
+MASS|7.3-58
```

Comment:
Where does this come from?

Reply:
I updated the PR description:

> Needed to pin packages `Matrix` and `MASS` to older versions which are still compatible with R 4.3

The latest versions of those packages require R 4.4.

Reply:
can you add this as comment to the package files

Reply:
Done